docs/connector-development/connector-builder-ui/incremental-sync.md

```diff
@@ -94,14 +94,25 @@ In some cases, it's helpful to reference the start and end date of the interval
 
 ## Incremental sync without time filtering
 
-Some APIs do not allow filtering records by a date field, but instead only provide a paginated "feed" of data that is ordered from newest to oldest. In these cases, the "API time filtering capabilities" option needs to be set to "No filter". As they can't be applied in this situation, the "Inject start time into outgoing HTTP request" and "Inject end time into outgoing HTTP request" options as well as the "Split up interval" option are disabled automatically.
+Some APIs do not allow filtering records by a date field, but instead only provide a paginated "feed" of all available. In these cases, the "API time filtering capabilities" option needs to be set to "No filter (data feed)". As they can't be applied in this situation, the "Inject start time into outgoing HTTP request" and "Inject end time into outgoing HTTP request" options as well as the "Split up interval" option are disabled automatically.
```

Comment:
let's update the doc in a separate PR since we'll need to update the builder separately

Reply:
reverted